tests/FreeDraw.test.js

```diff
@@ -86,7 +86,7 @@ test('It should be able to create polygons;', t => {
 
     // Ensure the expected functions are invoked.
     t.truthy(map.addLayer.called);
-    t.is(map.latLngToLayerPoint.callCount, polygon.length - 1);
+    t.is(map.latLngToLayerPoint.callCount, 2*(polygon.length - 1));
```

Comment:
Hey,
I have done this change because `latLngToLayerPoint` is being called **double**  number of times . 
One which happens in `fillPolygon() `and one in `isIntersectingPolygon()`  in `createFor()` . Thanks!